app/src/main/java/com/sun/basic_japanese/data/source/NHKLessonDataSource.kt

```diff
@@ -0,0 +1,9 @@
+package com.sun.basic_japanese.data.source
+
+import com.sun.basic_japanese.data.model.NHKLessonsResponse
+
+interface NHKLessonDataSource {
+    interface Local : NHKLessonDataSource {
```

Comment:
Interface NHKLessonDataSource có phương thức gì cần phải kế thừa từ Local này nhỉ ?
```suggestion
    interface Local {
```